beefy-gadget/src/keystore.rs

```diff
@@ -76,6 +85,20 @@ impl BeefyKeystore {
 		Ok(sig)
 	}
 
+	#[allow(dead_code)]
```

Comment:
I guess it's dead code because it's going to be used in future PR? I'd rather avoid doing anything like that - why can't we add this in the future PR itself? It's really tiny and I fear that if the future PR doesn't end up being created we will be left with this forever.

Reply:
I agree. There is a bit of a story behind this PR, however. If you look at the sequence of the commits, the PR was intended to implement and actually apply `public_keys()`. While writing a unit test for `public_keys()` it turned out that the testing key store is different from `LocalKeystore` to a point of being detrimental.

So, this PR changed it's nature and became a PR to get rid of the testing key store and use `LocalKeystore` instead. I really did not wanted to use `public_keys()` without knowing it works. On the other hand, I didn't wanted to bloat the PR any more by actually using `public_keys()`, either.

Guess, I'm asking for a waiver here on the promise that the "upcoming" PR will be the **next** PR.